package.json

```diff
@@ -14,7 +14,7 @@
     "grunt-contrib-copy": "0.6.0",
     "grunt-contrib-jshint": "0.11.0",
     "grunt-contrib-requirejs": "0.4.4",
-    "grunt-contrib-sass": "0.8.1",
+    "grunt-sass": "git://github.com/sindresorhus/grunt-sass.git#master",
```

Comment:
Only temporary until they release their proper tagged version, should be out later today.